source/eitri/eitri.py

```diff
@@ -44,43 +52,121 @@
 In the end, the file ".nav.lz4" is often used as input coverage data for Kraken.
 """
 
-logging.basicConfig(level=logging.DEBUG)
+
+logger = logging.getLogger("eitri")  # type: logging.Logger
+
+
+def is_file(file_path):
+    if not os.access(file_path, os.R_OK):
+        raise argparse.ArgumentTypeError("'" + file_path + "' is not a valid file path")
+    return file_path
+
+
+def is_folder(folder_path):
+    if not os.path.isdir(folder_path):
+        raise argparse.ArgumentTypeError("'" + folder_path + "' is not a valid folder path")
+    return folder_path
+
+
+def is_file_exist(file_path):
```

Comment:
'is_file_exist' isn't correct english. Also, it checks more than if the file exists.
```suggestion
def is_valid_file(file_path):
```

Reply:
Ok, done